target/linux/ath79/image/generic.mk

```diff
@@ -594,19 +647,22 @@ endef
 
 define Device/netgear_ex6400
   $(Device/netgear_ex7300_ex6400)
-  DEVICE_TITLE := NETGEAR EX6400
+  DEVICE_VENDOR := Netgear
```

Comment:
NETGEAR seems to always refer to itself with all-capital letters.

Reply:
Think about exploiting Device/netgear_ath79 in common-netgear.mk as you did with TP-Link and ubnt.
If you go that we, note that you need to override the On Network in tiny-netgear.mk (so just keep the DEVICE_VENDOR there, and remove it for all other Netgear devices).